app/src/main/res/values/strings.xml

```diff
@@ -50,5 +50,6 @@
   <string name="appearance">Appearance</string>
   <string name="advanced">Advanced</string>
   <string name="permissions_grant">Grant Permissions</string>
-  <string name="permissions_grant_body">In order for Aliucord Manager to work properly, you need to grant file permissions. Since this device is above Android 10, MANAGE_EXTERNAL_STORAGE is required to access ~/Aliucord, where shared data is stored.</string>
+  <string name="permissions_manage_storage_body">In order for Aliucord Manager to work properly, you need to grant storage permissions. Since this device is above Android 10, MANAGE_EXTERNAL_STORAGE is required to access ~/Aliucord, where shared data is stored.</string>
```

Comment:
Why does this need to be separate?
They are both file perms, one just looks slightly different. The user doesn't care about the specifics, they just want the app to work

Reply:
fair point